templates/documentation/cmip6_downscaled.html

```diff
@@ -0,0 +1,149 @@
+  1965&ndash;2100 and its extent covers most of Alaska and parts of western Canada. A 6-model average is also available,
+  consisting of an equal-weighted mean of the KACE-1-0-G, MIROC6, MPI-ESM1-2-HR, MRI-ESM2-0, NorESM2-MM, and TaiESM1
+  models.
+</p>
+
+<h3>Service endpoints</h3>
+
+<h4>CMIP6 downscaled daily data</h4>
+
+<p>
+  Query data for provided models, scenarios, and variables. These variables are defined <a
+    href="#cmip6-variable-information">below</a>.
+</p>
+
+<table class="endpoints">
+  <thead>
+    <tr>
+      <th class="endpoint-label">Endpoint</th>
+      <th class="endpoint-url">Example URL</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Daily values at a given latitude and longitude</td>
+      <td style="white-space: nowrap;">
+        <a href="/cmip6_downscaled/point/61.5/-147?models=6ModelAvg&scenarios=ssp585&vars=tasmax">
+          /cmip6_downscaled/point/61.5/-147?models=6ModelAvg&scenarios=ssp585&vars=tasmax</a>
+      </td>
+    </tr>
+  </tbody>
+  <tfoot>
+    <tr>
+      <td colspan="2">
+        Multiple models can be requested by appending a comma-separated string
+        like <code>?models=&lt;model_1&gt;,&lt;model_2&gt;</code>.<br />
+        Multiple scenarios can be requested by appending a comma-separated
+        string like <code>?scenarios=&lt;scenario_1&gt;,&lt;scenario_2&gt;</code>.<br />
+        Multiple variables can be requested by appending a comma-separated
+        string like <code>?vars=&lt;var_id_1&gt;,&lt;var_id_2&gt;</code>.<br />
+        CSV output is also available by appending <code>?format=csv</code> to
+        the URL.<br />
+        Chaining of multiple arguments is supported (e.g.,
+        <code>?vars=tasmax,tasmin&format=csv</code>).
+      </td>
+    </tr>
+  </tfoot>
+</table>
+
+<h4>Output</h4>
+
+<p>Results from point queries will look like this:</p>
+
+<pre>
+{
+  "5ModelAvg": {
+    "ssp585": {
+      "2015-01-01": {
+        "tasmax": -13
+      },
+      "2015-01-02": {
+        "tasmax": -13
+      },
+      "2015-01-03": {
+        "tasmax": -13
+      },
+      "2015-01-04": {
+        "tasmax": -14
+      },
+      ...
+    },
+    ...
+  },
+  ...
+}
+
+
+</pre>
+
+<p>The above output is structured like this:</p>
+
+<pre>
+{
+  &lt;model&gt;: {
+    &lt;scenario&gt;: {
+      &lt;year-month-day&gt;: {
+        &lt;var_id&gt;: daily value,
+        ...
+      },
+      ...
+    },
+    ...
+  },
+  ...
+}
+</pre>
+
+<h3 id="cmip6-variable-information">CMIP6 Variable Information</h3>
+
+<p>
+  Below is the list of CMIP6 variables that are found in this endpoint. For
+  complete variable definitions, see the official CMIP6 documentation
+  <a href="https://wcrp-cmip.org/cmip-model-and-experiment-documentation/">here</a>.
+</p>
+
+<dl>
+  <dt>pr</dt>
+  <dd>
+    Precipitation, including both liquid and solid phases (kg m<sup>-2</sup> s<sup>-1</sup>)
```

Comment:
Double check the units here vs. what is the CSV output, etc.

Reply:
Good call here, and fixed in a4f4ad5 to be "mm/day" (to be precise). The CSV metadata line says:

> pr is the daily precipitation in mm.

Which is a more verbose way to say the same thing, but they are both in agreement now.